include/boost/geometry/geometries/concepts/polygon_concept.hpp

```diff
@@ -115,9 +116,9 @@ class ConstPolygon
             ring_const_type ce = traits::exterior_ring<const_polygon_type>::get(*cpoly);
             interior_const_type ci = traits::interior_rings<const_polygon_type>::get(*cpoly);
 
-            boost::ignore_unused_variable_warning(ce);
-            boost::ignore_unused_variable_warning(ci);
-            boost::ignore_unused_variable_warning(cpoly);
+            boost::ignore_unused(ce);
+            boost::ignore_unused(ci);
+            boost::ignore_unused(cpoly);
```

Comment:
One function call.

Reply:
I skipped those as the list of argument is quite long

```
#ifdef BOOST_NO_CXX11_VARIADIC_TEMPLATES
            boost::ignore_unused(e, i, ce, ci, poly, cpoly);
#else
            boost::ignore_unused(e, i, ce, ci);
            boost::ignore_unused(poly, cpoly);
#endif
```

I'll replace with two calls

Reply:
Right, the emulation supports up to 5 arguments right now. So it's ok to call twice. I'd remove `#ifdef` because it needlessly complicates the code. With it it's 6 lines, the same number of lines as if you called once per variable.

Reply:
Yes, it's called twice now. The PR has been updated.

Reply:
Actually, this is the last place where one function could be called. Two calls we're talking about are above.

Reply:
Yes, it's fixed.